discovery-provider/src/utils/get_all_other_nodes.py

```diff
@@ -96,3 +98,18 @@ async def fetch_results():
             )
 
     return all_other_nodes, all_other_wallets
+
+
+def get_all_other_nodes_cached(redis) -> Tuple[List[str], List[str]]:
+    """
+    Attempts to get the number of discovery nodes from redis
+    if that doesn't exist it will fetch using `get_all_other_nodes()`
+    and set the result in the cache with an expiration of 1 hour
+    """
+
+    cached_nodes = get_json_cached_key(redis, ALL_OTHER_NODES_CACHE_KEY)
+    if cached_nodes:
+        return cached_nodes
+    nodes = get_all_other_nodes()
```

Comment:
it's kinda silly, also not blocking, but technically i think this endpoint could probably return *this* node too.

Reply:
ah good catch, added